app/controllers/training_and_support_controller.rb

```diff
@@ -12,4 +12,32 @@ def show
 
     @provider_name_key = @provider.name.downcase.gsub(/ /, "_")
   end
+
+  def update
+    @participant = current_user.participant_profile
+
+    if params[@participant.model_name.param_key].blank?
+      @participant.errors.add :viewed_guidance, "Select if you would like a brief summary of the programme"
+      render :guidance_question and return
+    end
+
+    @participant.update!(show_guidance_speedbump: false)
```

Comment:
I wonder if we should do the same in `show` action. After all, the intent is that the uses rees the guidance before they go to materials. If they happen to see them via a link on the dashboard, that probably counts, right?

Reply:
In short, we still want to show them the speedbump.

A notes been put on the ticket to say - a user may already click on the guidance separately, ahead of seeing the speedbump. However, we will show them the speedbump guidance regardless of whether they have independently clicked on the guidance straight from the dashboard or not.

Reply:
That... feels awkward. I would double check that this is a product intent, rather than a perceived technology limitation.

Reply:
No problem, i'll raise it at stand up